front/components/Logo/index.tsx

```diff
@@ -69,6 +69,7 @@ const Logo = ({ type = 'classic', size = null, className }: LogoProps): JSX.Elem
       <a className={classes.link}>
         <div className={classes.logo}>
           <Image
+            alt={`${type}-${size} c3pm logo`}
```

Comment:
I guess `c3pm logo` should be sufficient, right? I don't think screen reader users really care to know the type and size of the logo